polartools/diffraction.py

```diff
@@ -473,6 +475,7 @@ def load_series(
         )
 
     folder = kwargs.pop("folder", "")
+    scale = kwargs.pop("scale", [None, None])
```

Comment:
why don't you add "scale" as an explicit keyword argument? It can be hard for a user to figure out this option exists otherwise.